src/components/StudentPortfolio/ExpectationBox/ExpectationBox.tsx

```diff
@@ -18,7 +18,6 @@ export const ExpectationBox = () => {
    return (
       <>
          <SubtitlesSection text={expectation} />
-
```

Comment:
Please use linter and pretier to remove this emply lines. It is configured in project, but maybe its disabled in your local.